docs/kit/operation-view/installation_guide.md

```diff
@@ -0,0 +1,116 @@
+
+The deployment should then be modified to expect the Vault API at `http://localhost:8200`.
+The token should be set to a garbage value, as the agent will overwrite it with the real token.
+
+Setup
+------------
+1. Following mandatory properties will be required to fully configure and run the Connector in step 4 for use with Hashicorp Vault and PostgreSQL:
+    - `daps.url`: the URL of the DAPS service (e.g. https://daps1.int.demo.catena-x.net)
+    - `daps.clientId`: the client ID of the Connector (syntax: SKI:AKI)
+    - `vault.hashicorp.enabled` to `true`
+    - `vault.hashicorp.url`: the URL of the Hashicorp Vault service
+    - `vault.hashicorp.token`: the token of the Hashicorp Vault service to access the secrets
+    - `postgresql.enabled` to `true`
+    - `postgresql.jdbcUrl`: JDBC URL of the PostgreSQL database
+    - `postgresql.username`: username of the PostgreSQL database
+    - `postgresql.password`: password of the PostgreSQL database
+    - `controlplane.endpoints.authKey`: auth key of the control plane
+    - `backendService.httpProxyTokenReceiverUrl`: URL of the HTTP proxy token receiver service
+2. Further Connector specific properties can be configured:
+    - `internationalDataSpaces.id`: the id of the IDS connector
+    - `internationalDataSpaces.description`: the description of the IDS connector
+    - `internationalDataSpaces.title`: the title of the IDS connector
+    - `internationalDataSpaces.maintainer`: the maintainer of the IDS connector
+    - `internationalDataSpaces.curator`: the curator profile of the IDS connector
+3. Setup secrets in [Hashicorp](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-extensions/hashicorp-vault/README.md) Vault, add the following secrets:
+    - `transfer-proxy-token-signer-public-key`: key `content`, key-value `-----BEGIN PUBLIC KEY-----...` (2048 bits+)
+    - `transfer-proxy-token-signer-private-key`: key `content`, key-value `-----BEGIN RSA PRIVATE KEY-----...` (2048 bits+)
+    - Connector certificate PEM as `daps-public-key`: key `content`, key-value `-----BEGIN CERTIFICATE-----...`
+    - private-key PEM as `daps-private-key`: key `content`, key-value `-----BEGIN PRIVATE KEY----- ...`
+    - `transfer-proxy-token-encryption-aes-key`: key `content`, aes-key as key-value
+4. Install the Tractus-X EDC (please replace the angle brackets data with the information from step 1):
+   ```bash
+   helm repo add tractusx-edc https://eclipse-tractusx.github.io/charts/dev
+   helm install edc tractusx-edc/tractusx-connector --version 0.3.3 --set vault.hashicorp.enabled=true --set postgresql.enabled=true --set vault.hashicorp.url=<YOUR URL> --set vault.hashicorp.token=<YOUR TOKEN> --set daps.clientId=<YOUR ID> --set controlplane.endpoints.management.authKey=<YOUR AUTHKEY> --set postgresql.username=<YOUR USERNAME> --set postgresql.password=<YOUR PW> --set postgresql.jdbcUrl=<YOUR JDBC URL> --set backendService.httpProxyTokenReceiverUrl=<YOUR URL> --set daps.url=<YOUR URL>
```

Comment:
better to collect the values in a dedicated `*.yaml` file to keep the command easier to type. At the very least, please use shell line breaks